CHANGELOG.txt

```diff
@@ -1,9 +1,36 @@
 
+v6.6
+(01/04/18)
+
+Image additions:
+Nintendo Switch: DietPi Image now available, includes GPU, Kodi and RetroPie support! Many thanks to Mrs April and Mr Fall.
+
+Changes / Improvements / Optimizations:
+General | Removed minutely running "make_nas_processes_faster" cron job, which is present on some device base images, without any effect on raw DietPi: https://github.com/Fourdee/DietPi/issues/1654
```

Comment:
@Fourdee 

This line (nr. 9) is duplicate with line nr. 17 in this changelog.txt and as well here:

https://github.com/Fourdee/DietPi/blob/testing/CHANGELOG.txt#L32
https://github.com/Fourdee/DietPi/blob/testing/CHANGELOG.txt#L40

Not a big deal, but reading is a little bit bumpy. 😄

Reply:
@k-plan 
Good spot!